index.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <!-- If the browser is lower than or equal to IE 9, redirect to the old page.
+        TODO verificar outros navegadores
+        TODO colocar a página correta de redirecionamento (quando for para produção, 
+            criar um link para a página antiga e.g. matrusp/old, matrusp/v1) -->
+    <!--[if lte IE 9 ]>
+        <meta http-equiv="refresh" content="0; url=http://bcc.ime.usp.br/matrusp/" />
+    <![endif]-->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <title>MATRUSP</title>
+    
+    <link rel="stylesheet" href="styles/css/application.css">
+    
+</head>
+
+<body>
+    <div id="page-container">
+        <header>
+            <span id="logo">MATRUSP</span>
+            <ul id="icons">
+                <li class="icon" data-description="Exportar para o calendario do Google">
```

Comment:
Localization: Please extract all strings to a common place, a 'labels.pt-br.json' file for instance. We then read such file and replace any labels with their corresponding localized strings.